apps/src/lab2/views/Lab2Wrapper.tsx

```diff
@@ -48,8 +48,9 @@ export interface Lab2WrapperProps {
 const Lab2Wrapper: React.FunctionComponent<Lab2WrapperProps> = ({children}) => {
   const isLoading: boolean = useSelector(isLabLoading);
   const isPageError: boolean = useSelector(hasPageError);
-  const {isBlockedAbuse, projectSharingDisabled} = useAppSelector(
-    state => state.lab
+  const isBlockedAbuse = useAppSelector(state => state.lab.isBlockedAbuse);
+  const projectSharingDisabled = useAppSelector(
+    state => state.lab.projectSharingDisabled
```

Comment:
Tangentially related Lab2 optimizations - these drastically cut down on re-renders because `validationState`, which is part of `lab2redux`, is re-created on every Music Lab playback tick (something we could probably optimize separately). If we select `state.lab` here, this hook re-executes every tick because `validationState` is changing, even though we only actually cared about `isBlockedAbuse` and `projectSharingDisabled`. By instead only selecting those values explicitly, we only re-render when those values have actually changed.